### PR TITLE
Task-5-Article-view

### PR DESCRIPTION
**Pull request by Ziyardeen**
A single article view implemented

### CODE REVIEW

File: src/components/ArticleView.jsx [hunk 1]
```diff
@@ -0,0 +1,42 @@
+ import React, { useEffect, useState } from 'react'
+import { useParams } from 'react-router-dom'
+import { fetchArticleById } from '../../api/requests'
+ 
+ 
+ const ArticleView = () => {
+   
+    const {article_id} = useParams()
+    const [article,setArticle] = useState({})
+    console.log(article_id)
```
Comment (nc-bela-bertalan) [inline src/components/ArticleView.jsx:10]: Please remove all debugging logs.

File: src/components/ArticleView.jsx [hunk 2]
```diff
@@ -0,0 +1,42 @@
+ import React, { useEffect, useState } from 'react'
+import { useParams } from 'react-router-dom'
+import { fetchArticleById } from '../../api/requests'
+ 
+ 
+ const ArticleView = () => {
+   
+    const {article_id} = useParams()
+    const [article,setArticle] = useState({})
```
Comment (nc-bela-bertalan) [inline src/components/ArticleView.jsx:9]: Please add a state and indicate loading for the user.

File: api/requests.js [hunk 3]
```diff
@@ -27,3 +27,12 @@ export function fetchRandomArticles() {
       return random;
     });
 }
+
+export function fetchArticleById(article_id) {
+  return API.get(`/articles/${article_id}`).then(({ data }) => {
+    // console.log(data.article);
```
Comment (nc-bela-bertalan) [inline api/requests.js:33]: Please remove all commented-out code.

**Review by nc-bela-bertalan — COMMENTED** [review]
Hi Ziyardeen, please see my comments.

File: src/components/ArticleView.jsx (hunk 2 above)
Comment (Ziyardeen) [reply src/components/ArticleView.jsx:9]: Done. I have implemented that in my current working branch.

**Pull request merged**